src/20_Components/amp-bind.html

```diff
@@ -259,6 +259,17 @@
     </amp-img>
   </section>
 
+    <!-- ## Refresh state -->
+
+    <!-- `amp-state` supports the `refresh` action. This can be useful in many cases, for example the initial URL loading fails
+       and you want the user to load the state again.
+       For a complete list of actions, see [here](https://www.ampproject.org/docs/interaction_dynamic/amp-actions-and-events).
+       Clicking the button below will refresh and refetch the json in amp-state. -->
+
+    <button class="ampstart-btn caps m1" on="tap:remoteAnimals.refresh">
```

Comment:
It'd be cool to demo this somehow, e.g. have a document that always returns the current time.

Reply:
ok, will take a look